### PR TITLE
GA and MSCC cookie

### PR DESCRIPTION
**Pull request by octref**
This has been approved on cookie portal and is good to go.

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
 {% include head.html %}
 
 <body>
-
+    {% include cookie_notice.html %}
     {% include topnav.html %}
```
Comment (auchenberg) [inline _layouts/default.html]: Hallo

Comment (auchenberg) [reply _layouts/default.html]: **halloo**

**Comment by octref** [comment]
@dbaeumer @auchenberg Anything blocking this?

**Comment by auchenberg** [comment]
LGTM

**Comment by dbaeumer** [comment]
Fine with me. Feel free to merge it in. You should have both commit rights to do so.

**Pull request merged**